base_partner_merge/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Deduplicate contact (No CRM)
+============================
+
+This module installs the deduplicate wizard from CRM without the dependency on CRM.
+
+If you have CRM installed you don't need this module.
+
+
+Installation
+============
+
+To install this module, you need to have crm module present (but not installed).
+This is because we reuse the existing wizard without installing CRM.
+
+
+Usage
+=====
+
+You can access the wizard by going in a view list of contacts select one or multiple
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/11.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/partner-contact/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Author
+------
+
+Yannick Vaucher
```

Comment:
add email `<yannick.vaucher@camptocamp.com>` and "Do not contact contributors directly about support or help with technical issues." see https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst